### ISSUE TITLE
Flickering when starting new search with /

### ISSUE THREAD

**Issue by balta2ar** [haya14busa/incsearch.vim#60]
When I start new search (`map /  <Plug>(incsearch-forward)`), current highlighting is cleared immediately and screen flickers as if `<C-l>` has been pressed. I'm using `vim -g` (no problems in console version though).
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Dec  1 2014 12:50:52)
Included patches: 1-540

In .vimrc I have `set novisualbell` and `set lazyredraw` but they don't visually change anything.

Also, highlighting is cleared as soon as I type `/`. Is it possible to clear it only after I entered at least one symbol?

**Comment by haya14busa** [comment]
> When I start new search (map / <Plug>(incsearch-forward)), current highlighting is cleared immediately and screen flickers as if <C-l> has been pressed. I'm using vim -g (no problems in console version though).

Thanks for the report. it's related with this library issue #107. (taked in Japanese but the issue is same.)

> Also, highlighting is cleared as soon as I type /. Is it possible to clear it only after I entered at least one symbol?

It's possible. I'll consider to change the behavior, thanks :+1:

**Comment by haya14busa** [comment]
I fixed it. Please update incsearch.vim. :tada:

**Comment by balta2ar** [comment]
Yay! It's smooth now! Thank you very much, @haya14busa!

**Comment by balta2ar** [comment]
> > Also, highlighting is cleared as soon as I type /. Is it possible to clear it only after I entered at least one symbol?

> It's possible. I'll consider to change the behavior, thanks :+1:

What are your thoughts on this part?

**Comment by balta2ar** [comment]
More generally, highlighting is turned off if I press Esc, which is confusing. If I canceled my search, I'd like my previous highlighting to be restored.

**Comment by haya14busa** [comment]
Sorry I forget highlighting related issue. I create new issue #74
